src/layers/mlp_llama.cpp

```diff
@@ -54,6 +54,38 @@ void invokeMLPLLaMA(DataType dt, int numTokens, int hiddenSize, int intermediate
             llama_mlp = it_created->second;
         }
 
+        ctx->resize(1, numTokens, 0);
+        llama_mlp->forward(ctx, (float *)const_cast<void *>(input), (float *)output, inputStride, outputStride, false);
+    } else if (dt == DataType::fp16) {
```

Comment:
Can these duplicated or similar code lines for fp16 be reused for the bf16 section?

Reply:
Thinked, and the answer is no.